prensa_latas/src/components/QrReader.css

```diff
@@ -0,0 +1,125 @@
+.qr-reader {
+    width: 100;
+    height: 100;
+}
+
+.qr-reader section {
+  position: static !important;
+}
+
+.qr-reader section div {
+    width: 50vw !important;
+    height: 50vw !important;
+    left: 25% !important;
+    top: 50% !important;
+    margin-top: -25vw;
+    box-shadow: none !important;
+    border: none !important;
+}
+
+.corner-border {
+  width: 50vw;
+  height: 50vw;
+  left: 0;
+  top: 0;
+  margin-top: -25vw;
+  position: absolute;
+  border-left: 25vw solid rgba(0, 0, 0, 0.3);
+  border-top: 50vh solid rgba(0, 0, 0, 0.3);
+  border-right: 25vw solid rgba(0, 0, 0, 0.3);
+  border-bottom: 50vh solid rgba(0, 0, 0, 0.3);
+}
+
+.back {
+  height: 90px;
+  width: 90px;
+  bottom: -3.5vw;
+  left: -2.5vh;
+  position: absolute;
+  background-color: #00695C;
+  border-radius: 50%;
+  border: none;
+}
+
+.arrow {
+  border: solid rgb(235, 224, 224);
+  border-width: 0 6px 6px 0;
+  display: inline-block;
+  padding: 12%;
+  top: 25%;
+  margin-left: 25%;
+}
+
+.arrow-left {
+  transform: rotate(135deg);
+}
+
+.popup-error {
+  position: absolute;
+  top: 50vh;
+  left: 10vw;
+  width: 80vw;
+  height: 50vw;
+  margin-top: -25vw;
+  z-index: 1;
+  background-color: #E2F1F8;
```

Comment:
### Problem

Here you should import the color from a file that contains all colors in a JS object. At `dev`, this file is `src/style/colors.js`.

### Solution

To avoid problems with CSS importing and manipulation, a solution is using a JS object with style properties instead of a CSS file. Take a look at `src/components/NavBar/style.js` (`dev` branch) to have an example.

Just be careful with some little differences, for example, camelCase is used instead of hyphen separated, and every value that is not purely a number, like `100%`, is a string instead of its literal value.

To use this object, just import it on your `index.js` and use the `style` prop at each tag. Take a look at `src/components/NavBar/index.js` (`dev` branch) to have an example.